todo/src/components/TodoItem.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { changeTodo } from '../actions';
+
+class TodoItem extends React.Component {
+  state = {
+    isDone: this.props.todoItem.isComplete,
+    item: {
+      id: this.props.todoItem.id,
+      task: this.props.todoItem.task,
+      isComplete: this.props.todoItem.isComplete
+    }
+  }
+
+  onClick = event => {
+    event.preventDefault();
+    this.setState({
+      isDone: !this.state.isDone
+    });
+    this.props.changeTodo(this.state.item);
+  }
+
+  render() {
+    return(
+      <li onClick={this.onClick} className={`${this.state.isDone ? 'done' : 'todo'}`}>{this.props.todoItem.task}</li>
```

Comment:
Also wondering why you chose to read directly from state on the `className` here as opposed to the redux store, I suspect it is because your component was not re-rendering when the redux store was updated.